components/test_raftstore/src/cluster.rs

```diff
@@ -127,6 +127,10 @@ impl<T: Simulator> Cluster<T> {
         self.cfg.server.cluster_id
     }
 
+    pub fn pre_start(&mut self) -> result::Result<(), Box<dyn StdError>> {
+        self.cfg.validate()
```

Comment:
can we call it directly in `start`?

Reply:
The idea is to check config before `create_engines`